apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -3,72 +3,130 @@
+            {% if current_page.publisher_is_draft %}
+              {% if subject.public_extension %}
+                <li class="course-detail__content__subjects__item">
+                  {{ subject.public_extension.extended_object.get_title }}
+                </li>
+              {% else %}
+                <li class="course-detail__content__subjects__item--draft">
+                  {{ subject.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current course page is the published version, show only the subjects that are published #}
+            {% elif subject.public_extension %}
+              <li class="course-detail__content__subjects__item">
+                {{ subject.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% empty %}
+            <li class="course-detail__content__subjects--empty">
+              {% trans "No associated subjects" %}
             </li>
-          {% endif %}
-        {# If the current page is the published version, show only the organizations that are published #}
-        {% elif organization.public_extension %}
-          <li class="course-detail__organizations__item">
-            {{ organization.public_extension.extended_object.get_title }}
-          </li>
-        {% endif %}
-      {% endfor %}
-    </ul>
-
-    <div class="course-detail__active-session">
+          {% endfor %}
+        </ul>
+      </div>
+
+      <div class="course-detail__content__organizations">
+        <h2>{% trans 'Organizations' %}</h2>
+        <ul class="course-detail__content__organizations__list">
+          {% for organization in course.organizations.all %}
+            {# If the current page is a draft, show draft organizations with a class annotation for styling #}
+            {% if current_page.publisher_is_draft %}
+              {% if organization.public_extension %}
+                <li class="course-detail__content__organizations__item">
+                  {{ organization.public_extension.extended_object.get_title }}
+                </li>
+              {% else %}
+                <li class="course-detail__content__organizations__item--draft">
+                  {{ organization.extended_object.get_title }}
+                </li>
+              {% endif %}
+            {# If the current page is the published version, show only the organizations that are published #}
+            {% elif organization.public_extension %}
+              <li class="course-detail__content__organizations__item">
+                {{ organization.public_extension.extended_object.get_title }}
+              </li>
+            {% endif %}
+          {% endfor %}
+        </ul>
+      </div>
+
+    <div class="course-detail__content__information">
+      {% placeholder "course_information" %}
+    </div>
+
+    <div class="course-detail__content__license">
+      <h2>{% trans 'License' %}</h2>
+      {% placeholder "course_license" or %}
```

Comment:
What is the license under which this course is published?

Reply:
After checking, in fact we need 2 placeholders:
```html
<h2>{% trans 'License' %}</h2>
<h3>{% trans 'License for the course content' %}</h3>
{% placeholder "license_course_content" or %}
<p>{% trans 'What is the license for the course content?' %}</p>
{% endplaceholder %}
<h3>{% trans 'License for the content created by course participants' %}</h3>
{% placeholder "license_course_participation" or %}
<p>{% trans 'What is the license for the content created by course participants?' %}</p>
{% endplaceholder %}
```

For each placeholder, the person editing the course page can only add a license plugin as defined here: https://github.com/openfun/richie/issues/273